src/test/java/tictactoe/GameOutcomeTest.java

```diff
@@ -0,0 +1,15 @@
+package tictactoe;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class GameOutcomeTest {
```

Comment:
Could you try and find a behaviour of your game where this is tested? It's currently testing an implementation detail (that the `GameOutcome` enum has a property that returns the string `TIE`). This is fragile, since it will fail if you ever decide to change logic which ought to be encapsulated within the enum.

Reply:
Good point. A place I can test this is in the `InputValidator` class. For example, instead of expecting the user's selection to be the same as `"h"` or `"human"`, I can compare it to `PlayerTypes.human.toString()`